src/runtime-rs/crates/resource/src/share_fs/utils.rs

```diff
@@ -59,6 +59,10 @@ pub fn get_host_rw_shared_path(sid: &str) -> PathBuf {
     Path::new(KATA_HOST_SHARED_DIR).join(sid).join("rw")
 }
 
+pub fn get_host_shared_path(sid: &str) -> PathBuf {
```

Comment:
suggest rename to get_host_sbs_shared_path.

Reply:
Maybe the name is too long? I think maybe other developers should know the share path is only for sandbox?

Reply:
fair enough